examples/regional_private_node_pool_oauth_scopes/network.tf

```diff
@@ -16,7 +16,7 @@
 
 module "gke-network" {
   source  = "terraform-google-modules/network/google"
-  version = ">= 4.0.1, < 5.0.0"
```

Comment:
I think we may want `>= 4.0.1` for min provider 4.x in networking but the max constraint can be lifted. I am not sure why we added the `< 5.0.0` but it was likely a bug in my script when I did https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1129 where it thought this was TPG constriant.

https://github.com/terraform-google-modules/terraform-google-network/releases/tag/v4.0.1

Reply:
agreed